### DISCUSSION TITLE
Motion Gyro+Accel

### DISCUSSION THREAD

**Discussion by Himoe2023** [gopro/labs#519] [Q&A]
I'm trying to use gopro labs to record on a Hero 11 Mini while riding a bike. Using the Motion detection via gyro+accel, does the camera need to be on before the recording will automatically start, or can it be off with the recording auto starting when it detects it's moving? Also, since biking requires starting from a stop, what should i set the sensitivities to so that the recording will start immediately when the bike starts moving, and stop after 5 seconds after it stops moving?

Thanks,

**Comment by dnewman-gpsw** [comment]
Camera will need to powered on, a gyro is not awake.  Set the sensitivity is a matter of experimentation.  I use this feature for drone video captures, I had to tweak to get a good setting.
